### ISSUE TITLE
Missing file or unneeded reference.

### ISSUE THREAD

**Issue by rdebath** [ClassiCube/MCGalaxy#697]
```
./MCGalaxy/MCGalaxy_.csproj:    <Compile Include="sharkbite.thresher\NameGenerator.cs" />
```

**Comment by UnknownShadow200** [comment]
I forgot to click save all before pushing commit, it will be fixed as part of next commit

**Comment by UnknownShadow200** [comment]
Resolved by 4be17cccc8db743d1474e2411df8507e8baf8b9d
